btree/btree.c

```diff
@@ -3328,40 +3328,43 @@ int calc_shift(int value)
  * btree_create_tick function is the main function used to create btree.
  * It traverses through multiple states to perform its operation.
  *
- * @param smop Represents the state machine operation
- * @return int64_t It returns the next state to be executed.
+ * @param smop     represents the state machine operation
+ * @return int64_t returns the next state to be executed.
  */
-
 int64_t btree_create_tick(struct m0_sm_op *smop)
 {
-	struct m0_btree_op    *bop = M0_AMB(bop, smop, bo_op);
-	struct m0_btree_oimpl *oi = bop->bo_i;
-	struct m0_btree_idata *data = &bop->b_data;
+	struct m0_btree_op    *bop    = M0_AMB(bop, smop, bo_op);
+	struct m0_btree_oimpl *oi     = bop->bo_i;
+	struct m0_btree_idata *data   = &bop->b_data;
 	int                    k_size = data->bt->ksize == -1 ? MAX_KEY_SIZE :
 					data->bt->ksize;
 	int                    v_size = data->bt->vsize == -1 ? MAX_VAL_SIZE :
 					data->bt->vsize;
 
-
 	switch(bop->bo_op.o_sm.sm_state)
 	{
```

Comment:
curly bracket should be on line 3344. Align rest of the code accordingly after the change.

Reply:
Ok.